File: Data Structures & Algorithms/two-integer-sum/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @param {number} target
     * @return {number[]}
     */
    twoSum(nums, target) {
        for(let j = 0; j<nums.length; j++) {
            for(let i=0; i < nums.length; i ++){
                if(j == i)
                    continue
                if(nums[j] + nums[i] === target){
                    return [j, i]
                }
            }
        }
    }
}

File: Data Structures & Algorithms/two-integer-sum-ii/submission-0.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        result = []
        for idx,num in enumerate(numbers):
            for n in range(len(numbers) -1, idx, -1):
                if num + numbers[n] == target:
                    result.append(idx +1)
                    result.append(n +1)
        return result